tests/integration/test_analysisd/test_rootcheck/conftest.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2015-2023, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+import time
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.constants.paths.sockets import QUEUE_DB_PATH
+from wazuh_testing.modules.modulesd import patterns
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils import callbacks
+from wazuh_testing.utils.manage_agents import remove_agents
+from wazuh_testing.utils.services import control_service
+from wazuh_testing.tools.db_administrator import DatabaseAdministrator
+from wazuh_testing.tools.simulators.agent_simulator import connect, create_agents
+
+def inject_rootcheck_events(agents):
```

Comment:
If you apply the previous changes suggested, this function won't be necessary. Just add the sleep in the tests.